.github/workflows/docker.yml

```diff
@@ -1,40 +1,75 @@
-name: docker
+name: Build
+
+# Controls when the workflow will run
 on:
   # Scheduled workflows run on the latest commit on the default or base branch
-  schedule:
-    - cron: '20 2 * * *' # Daily at 02:20
+  # schedule:
+  #   - cron: '20 2 * * *' # Daily at 02:20
 
   push:
+    branches:
+      - feature/docker-multi-arch-support
     tags:
       - v*
 
 permissions:
   contents: read # to fetch code (actions/checkout)
 
 jobs:
-  publish:
-    # restrict this job to base repo for now
+  build:
     if: github.repository == 'opf/openproject'
     runs-on: ubuntu-latest
-    env:
-      INPUT_BUILDOPTIONS: --pull
+    strategy:
+      matrix:
+        include:
+          - base_prefix:
+            platforms: linux/amd64
+          - base_prefix: ppc64le/
+            platforms: linux/ppc64le
+            bim_support: false
+          - base_prefix: arm64v8/
+            platforms: linux/arm64/v8
+            bim_support: false
     steps:
-      - uses: actions/checkout@v3
+      # Get the repository's code
+      - name: Checkout
+        uses: actions/checkout@v3
+      # https://github.com/docker/setup-qemu-action
       - name: Prepare docker files
         run: |
           cp ./docker/prod/Dockerfile ./Dockerfile
-      - name: Build & Push
-        id: build_and_push
-        uses: elgohr/Publish-Docker-Github-Action@master
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      # https://github.com/docker/setup-buildx-action
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
         with:
-          name: openproject/community
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-          tag_semver: true
-          cache: ${{ github.event_name != 'schedule' }}
-      - name: Test
```

Comment:
You removed the test step from the steps, can we enable this for all builds now?